roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -88,6 +88,7 @@ networkConfig:
   clusterNetworkCIDR: {{ openshift.master.sdn_cluster_network_cidr }}
   hostSubnetLength: {{ openshift.master.sdn_host_subnet_length }}
   networkPluginName: {{ openshift.common.sdn_network_plugin_name }}
+  serviceNetworkCIDR: {{ openshift.master.portal_net }}
```

Comment:
Based on discussion with @danwinship it won't be deprecated. I've added a comment to the template indicating they must be the same.